src/components/molecules/search-input/search-input.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Button from 'components/atoms/button';
+import TextInput from 'components/atoms/text-input';
+
+const SearchInput = () => (
```

Comment:
I was having trouble getting the atoms to work with the form elements so punted on fleshing this out but ideally, this would be the text input + button packaged as a single molecule. Things to do to make it extensible:
- accept name
- accept onChange handlers
- should bake in form errors
- needs to be a in flexible container